guides/InstallationAdminGuide.md

```diff
@@ -333,6 +333,8 @@ Once the server has been started, the success of the installation can be checked
 
 The openEQUELLA Server administration account is hidden from casual users and is displayed by entering a special URL created from the server’s base URL. 
 
+Note: The base URL is configured in `learningedge-config/mandatory-config.properties` as `admin.url`. If admin.url is changed adjust the instructions below accordingly.
+
 ### To open the Server administration account page
 1. Open a browser and enter the openEQUELLA address of the hosting server with 
 ‘/institutions.do?method=admin’ appended to the URL. (e.g. ‘http://equella.myinstitution.edu/logon.do’ would become ‘http://equella.myinstitute.edu/institutions.do?method=admin’).
```

Comment:
I would change this to:

Open a browser and enter the openEQUELLA address of the `admin.url` with 
‘/institutions.do?method=admin’ appended.

Reply:
:+1: